include/wifi_hal_radio.h

```diff
@@ -90,6 +90,16 @@ typedef struct
     UCHAR punct_acs_threshold; /**< Puncturing ACS threshold. */
 } __attribute__((packed)) wifi_radio_11be_puncturing_info_t;
 
+#define MAXNUMNONOPERABLECHANNELS 10
+#define MAXNUMOPERCLASSESPERBAND 20
```

Comment:
unreadable `MAXNUMOPERCLASSESPERBAND` -> `MAX_NUM_OPER_CLASSES_PER_BAND`

Reply:
Similar comment as above. 
1) Is used once in the code in https://github.com/rdkcentral/OneWifi/blob/main/source/utils/wifi_util.c function: update_radio_operating_classes
2) Used the earlier convention in the file described above.

Reply:
I will document it and update the pull request